apps/music/test/unit/player_view_test.js

```diff
@@ -0,0 +1,105 @@
+/* global PlayerView */
+
+'use strict';
+
+require('/js/ui/views/player_view.js');
+require('/shared/js/l10n.js');
+
+suite('Star Rating Bar Accessibility Test', function () {
+
+    var pv,
+    ratings;
+
+    suiteSetup(function () {
+       
```

Comment:
The empty lines are not an issue. You can be liberate with them. But they should not contain spaces in them. the only reason i spotted was that my editor (Sublime) highlights them and automatically removes (with Trailing Spaces add on). So for example on line 14 (which is fine to have empty) there are 7 spaces in the beginning of the line that need to be removed. The rest of the lines I mentioned have the same issue.